packages/@ember/-internals/runtime/lib/mixins/evented.js

```diff
@@ -120,10 +115,12 @@ export default Mixin.create({
     @method trigger
     @param {String} name The name of the event
     @param {Object...} args Optional arguments to pass on
+    @return this
     @public
   */
   trigger(name, ...args) {
     sendEvent(this, name, args);
+    return this;
```

Comment:
also added `return this` since all other methods have it

Reply:
Please revert this part, it’s unrelated to the main point of the PR and I don’t personally think we should encourage the chaining...